### PR TITLE
Subaru: enable alpha longitudinal for gen1 cars

### PR DESCRIPTION
**Pull request by jnewb1**
AEB is not preserved, PCB warning is shown on dash
https://github.com/commaai/openpilot/blob/26a67641d2e4a4b908239996c7a9b580c3813d4b/selfdrive/car/subaru/subarucan.py#L163

Other eyesight functionality is lost (high beam assist, etc) since eyesight will fault for not receiving the proper feedback signals
Vision only longitudinal

### CODE REVIEW

File: selfdrive/car/subaru/values.py [hunk 1]
```diff
@@ -93,6 +93,9 @@ class Footnote(Enum):
   GLOBAL = CarFootnote(
     "In the non-US market, openpilot requires the car to come equipped with EyeSight with Lane Keep Assistance.",
     Column.PACKAGE)
+  EXP_LONG = CarFootnote(
+    "Enabling longitudinal control will disable all EyeSight functionality, including AEB, LDW, and RAB.",
```
Comment (sshane) [inline selfdrive/car/subaru/values.py:97]:
```suggestion
    "Enabling longitudinal control (alpha) will disable all EyeSight functionality, including AEB, LDW, and RAB.",
```

**Comment by sshane** [comment]
If you think it is in a decent enough state to ship to master-ci/devel/forks, let's do it.

**Comment by jnewb1** [comment]
> If you think it is in a decent enough state to ship to master-ci/devel/forks (no fault lights on dash, decent actuation control, shows AEB light), let's do it.

only fault lights are from systems that are actually disabled (aeb, ldw, RAB, etc)
I've driven the outback for a few hours with eyesight disabled, actuation is pretty decent

**Comment by sshane** [comment]
make sure to update docs before mergin

**Pull request merged**